content/en-us/projects/managing.md

```diff
@@ -33,6 +33,37 @@ define project settings:
    This unlocks more tabs to define other project settings as
    described in [Game Settings](../studio/game-settings.md).
 
+## Cloud-Based Assets
```

Comment:
Just moving these two sections up from a bit lower down. Reading about assets and processes for testing naturally come before publishing in the typical workflow.